features/support/run.js

```diff
@@ -36,7 +37,7 @@ module.exports = function () {
     };
 
     this.runBin = (bin, options, env, callback) => {
-        let cmd = util.format('%s%s/%s%s%s', this.QQ, this.BIN_PATH, bin, this.EXE, this.QQ);
+        let cmd = path.resolve(util.format('%s/%s%s', this.BIN_PATH, bin, this.EXE));
```

Comment:
double quoted commands in `execFile` do not work for me on windows. I see the only reason to have double quotes around command is spaces in the path, but in http://stackoverflow.com/questions/16395612/unable-to-execute-child-process-exec-when-path-has-spaces stated that spaces must be escaped with backslashes. I did not check exactly this case, but think that `path.resolve` returns correctly escaped absolute path